### PR TITLE
Replace CPAN ratings site with redirects to metacpan

### PR DESCRIPTION
**Pull request by abh**

### CODE REVIEW

**Comment by abh** [comment]
@rspier I restarted the containers in kube and because the git:// change the old one doesn't sync anymore (oops), so it's running off this branch now. When merged I'll switch the site back to run off main.

File: lib/CPANRatings/Model/SearchCPAN.pm [hunk 1]
```diff
@@ -9,130 +9,133 @@ use URI::Escape ();
...
-      }
-      $data = \@results;
-
-      #use Data::Dump qw(pp);
-      #warn pp($data);
+        $data = $data->{data};
     }
     else {
-      $data = [];
+        my $url = "http://search.cpan.org/search?query=$query&mode=$mode&format=xml";
```
Comment (rspier) [inline lib/CPANRatings/Model/SearchCPAN.pm:35]: Does this API actually still work?

File: lib/CPANRatings/Model/SearchCPAN.pm [hunk 2]
```diff
@@ -9,130 +9,133 @@ use URI::Escape ();
...
 }
 
 sub search_distribution {
-  shift->_search('dist', @_);
+    shift->_search('dist', @_);
 }
 
 sub _search {
-  my ($self, $mode, $query) = @_;
+    my ($self, $mode, $query) = @_;
```
Comment (rspier) [inline lib/CPANRatings/Model/SearchCPAN.pm:23]: Looks like this introduced some unnecessary whitespace changes.

**Review by rspier — APPROVED** [review]

File: lib/CPANRatings/Model/SearchCPAN.pm (hunk 1 above)
Comment (abh) [reply lib/CPANRatings/Model/SearchCPAN.pm]: I thought it did because the site worked, but it's failing in a way that I will just leave it out instead.

File: lib/CPANRatings/Model/SearchCPAN.pm (hunk 2 above)
Comment (abh) [reply lib/CPANRatings/Model/SearchCPAN.pm]: yeah, it was whatever my editor did. I tried doing perl tidy on the files but that just made different changes... 🤷
